examples/sql_bank/main.go

```diff
@@ -26,14 +26,18 @@ import (
 
 	"github.com/lib/pq"
 
+	"github.com/cockroachdb/cockroach/security"
+	"github.com/cockroachdb/cockroach/security/securitytest"
+	"github.com/cockroachdb/cockroach/server"
 	"github.com/cockroachdb/cockroach/util/log"
 )
 
 const (
 	maxTransfer = 999
 	numAccounts = 999
-	concurrency = 25
+	concurrency = 5
 	aggregate   = true
+	usePostgres = false
```

Comment:
I'd rather align this completely with the KV bank app soon and do that all in the same change.